### PR TITLE
ServiceInsight SSL validation override document.

### PR DESCRIPTION
**Pull request by HEskandari**
Connects to https://github.com/Particular/ServiceInsight/issues/717

### CODE REVIEW

File: serviceinsight/self-signed-certificate-connection.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+reviewed: 2016-09-28
+summary: ServiceInsight refuses to connect to a ServiceControl instance running on SSL with self-signed certificate.
+component: ServiceInsight
+tags:
+- SSL
+- ServiceControl
+- Certificate
+---
+
+ServiceControl can be configured to run on a SSL, but if a self-signed certificate is chosen, ServiceInsight would not allow connections to that instance. While running a self-sign certificate is not recommended, ServiceInsight provides an override configuration entry to allow you to connect to such instances.
```
Comment (weralabaj) [inline serviceinsight/self-signed-certificate-connection.md]: run over SSL

Comment (weralabaj) [inline serviceinsight/self-signed-certificate-connection.md]: > can be configured to run on a SSL

run **over** SSL

Comment (weralabaj) [reply serviceinsight/self-signed-certificate-connection.md]: >  While running a self-sign certificate is not recommended

Should we explain why?

**Comment by SimonCropp** [comment]
this will need a rebase

**Comment by HEskandari** [comment]
I have marked this as WIP. It would require some changes as the configuration code has changed after the review by the SI maintainers.

File: serviceinsight/self-signed-certificate-connection.md (hunk 1 above)
Comment (adamralph) [reply serviceinsight/self-signed-certificate-connection.md]: Discussed with @Particular/serviceinsight-maintainers. We will hyperlink "running a self-sign certificate is not recommended".

**Comment by HEskandari** [comment]
@adamralph a very good point. I'll add it to SI menu under the 'Usage' section.

**Comment by weralabaj** [comment]
@HEskandari I see the linked issue has been closed https://github.com/Particular/ServiceInsight/issues/717, can we now complete this PR?

**Comment by MarcinHoppe** [comment]
Technically speaking it is HTTPS, which can run on top of SSL (no longer recommended and pretty much obsolete) or TLS (the current version).

**Comment by HEskandari** [comment]
@weralabaj Thanks for the reminder, I'll finalize it today.

**Comment by HEskandari** [comment]
@weralabaj fast forwarding a few broken builds, cause reasons....I think this can be merged now.

**Comment by adamralph** [comment]
@Particular/serviceinsight-maintainers should this document not mention that this feature only exists in 1.9.0 onwards? Also, I think we probably should not have closed https://github.com/Particular/ServiceInsight/issues/717 (and released 1.9.0) before this was merged.

**Comment by HEskandari** [comment]
I don't think we track feature by version on SI documents. As an example, some of the 'views' like Sequence Diagramn are not available on older version of SI. Not saying this is a bad idea though. It is just that it is more than just this page if we want to keep things consistent.

👍 on your comment about the release.

**Pull request merged**